scrapers/LustCinema.yml

```diff
@@ -0,0 +1,41 @@
+name: LustCinema
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lustcinema.com
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    common: &com
+      $cast: //div[contains(concat(' ',normalize-space(@class),' '),'relative px-1')]
+      $script: //script[contains(.,"window.__NUXT")]/text()
+    scene:
+      Title: &title //h1
```

Comment:
Doesnt work in following random scenes i tested
`https://lustcinema.com/movies/ridin-in-the-jungle`, `https://lustcinema.com/series/a-taste-of-kunst`, `https://lustcinema.com/movies/he-loves-me-in-collars-and-cuffs-2` . `https://lustcinema.com/movies/the-affairs-of-lidia`
I tried the below  and it seems to work (at least with the scenes mentioned above)
```yaml
      Title: //div[contains(@class,"my-3")]/img/@name
```